### PR TITLE
Add embed message customization for discord webhook and improve console logger

### PR DESCRIPTION
**Pull request by Reishimanfr**
Added a option to customize discord webhook messages via the `webhookMessages.yml` file (#65 ) and also improved how the console logger looks and works

### CODE REVIEW

File: sleepingSettings.yml
Comment (vincss) [inline sleepingSettings.yml:1]: Why is it deleted ?

Comment (Reishimanfr) [reply sleepingSettings.yml:1]: basically I messed up the original config file and it didn't pass the test so I deleted it lol
should I re-add it?

Comment (vincss) [reply sleepingSettings.yml:1]: Yes please :-)

File: src/sleepingSettings.ts [hunk 1]
```diff
@@ -47,6 +53,11 @@ export const DefaultSettings: Settings = {
   version: false,
 };
 
+export const DefaultMessages: MessagesType = {
```
Comment (vincss) [inline src/sleepingSettings.ts:56]: Specify that the messages are related to Discord in types and variable names.

File: tsconfig.json [hunk 2]
```diff
@@ -1,5 +1,5 @@
 {
-  "include": ["./src/**/*.ts"],
+  "include": ["./src/**/*.ts", "sleepingDiscord.ts"],
```
Comment (vincss) [inline tsconfig.json:2]: This shouldn't change

File: webhookMessages.yml [hunk 3]
```diff
@@ -0,0 +1,10 @@
+# This file contains the messages that are sent using the discord webhook if it's enabled
```
Comment (vincss) [inline webhookMessages.yml:1]: Put this setting in the same file as the other, I think it will only confuse people.

File: src/sleepingLogger.ts [hunk 4]
```diff
@@ -29,20 +35,32 @@ export const getLogger = () => {
...
 
+    const loggerFormat = format.combine(
+      format(info => ({ ...info, level: info.level.toUpperCase() }))(),
+      format.align(),
+      format.colorize({ all: true }),
+      format.prettyPrint({ colorize: true }),
+      format.simple(),
+      format.errors({ stack: true }),
+      format.splat(),
+      format.timestamp({ format: "HH:mm:ss" }),
```
Comment (vincss) [inline src/sleepingLogger.ts:46]: put back the date.

File: webhookMessages.yml (hunk 3 above)
Comment (Reishimanfr) [reply webhookMessages.yml:1]: I believe it's better to try and split stuff into separate files to not make a single, bloated file but sure!

Comment (vincss) [reply webhookMessages.yml:1]: For code, I agree but for settings it will confuse the user.

**Review by vincss — APPROVED** [review]

**Pull request closed without merging**